charts/node/templates/statefulset.yaml

```diff
@@ -130,16 +130,30 @@ spec:
             - containerPort: 30333
               name: p2p
               protocol: TCP
+          {{- if .Values.node.enableStartupProbe }}
+          # On startup, retry the connection to the /health endpoint every 10s for 5 min before killing the container
+          startupProbe:
+            failureThreshold: 30
+            periodSeconds: 10
+            # Use an exec command as Substate's default configuration only allow local network access to the RPC endpoint
+            exec:
+              command:
+              - curl
+              - "http://127.0.0.1:9933/health"
```

Comment:
what's the difference to using the [httpGet call](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes)?

Reply:
In case `--rpc-external` is not enabled on the node the httpGet call will fail as will be blocked by substrate as an external network call. There might be cases where users want to run nodes without this flag enabled, so we do this call from within the container itself where it will never be blocked.